opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -118,255 +145,122 @@ message MetricDescriptor {
-    // Corresponding values are stored in Int64DataPoint.
-    GAUGE_INT64 = 1;
-
-    // Floating point gauge. The value can go both up and down over time.
-    // Corresponding values are stored in DoubleDataPoint.
-    GAUGE_DOUBLE = 2;
-
-    // Histogram gauge measurement.
-    // Used in scenarios like a snapshot of time that current items in a queue
-    // have spent there.
-    // Corresponding values are stored in HistogramDataPoint. The count and sum of the
-    // histogram can go both up and down over time. Recorded values are always >= 0.
-    GAUGE_HISTOGRAM = 3;
-
-    // Integer counter measurement. The value cannot decrease; if value is reset then
-    // start_time_unix_nano should also be reset.
-    // Corresponding values are stored in Int64DataPoint.
-    COUNTER_INT64 = 4;
-
-    // Floating point counter measurement. The value cannot decrease, if
-    // resets then the start_time_unix_nano should also be reset.
-    // Recorded values are always >= 0.
-    // Corresponding values are stored in DoubleDataPoint.
-    COUNTER_DOUBLE = 5;
-
-    // Histogram cumulative measurement.
-    // Corresponding values are stored in HistogramDataPoint. The count and sum of the
-    // histogram cannot decrease; if values are reset then start_time_unix_nano
-    // should also be reset to the new start timestamp.
-    CUMULATIVE_HISTOGRAM = 6;
-
-    // Summary value. Some frameworks implemented Histograms as a summary of observations
-    // (usually things like request durations and response sizes). While it
-    // also provides a total count of observations and a sum of all observed
-    // values, it calculates configurable percentiles over a sliding time
-    // window.
-    // Corresponding values are stored in SummaryDataPoint.
-    SUMMARY = 7;
+    // INSTANTANEOUS is a metric whose values are measured at a particular
+    // instant.
+    //
+    // If an INSTANTANEOUS metric is reported over any time interval the
+    // values are representative of distinct measurements within the
+    // interval.
+    INSTANTANEOUS = 1;
+
+    // DELTA is a metric whose values are measured and successively added
+    // over a reporting interval. The reported value represents the change
+    // from the previous interval.
+    //
+    // Each reported value for a DELTA metric over a time interval is
+    // self-contained, the values do not depend on values in other time
+    // intervals. This is unlike a CUMMULATIVE metric where values are sums
+    // of all measured events in the time interval and all previously
+    // measured values of the metric.
+    //
+    // DELTA metrics are not guaranteed to be monotonic.
+    DELTA = 2;
+
+    // CUMULATIVE is a metric whose values are measured and successively
+    // added to the lifetime total of the metric.
+    //
+    // Each reported value for a CUMULATIVE metric is the sum of all
+    // measurements up to and including that one.
```

Comment:
I don't think this sentence is accurate, as it reads almost like a math sequence `V_n = Sum_{i=1..n-1} V_i`

Reply:
That formula is mostly correct based on my understanding of what we are trying to convey. Only thing I would add is that the latest measurement is included in the sum. I.E.

```
Value_n = Sum_{i=1...n} Measurement_i
```

Do you have a different way of understanding what a CUMULATIVE metric is, or maybe a better way of stating this?

Reply:
Does CUMULATIVE only applies to sums?

Your formula is still confusing (note that mine didn't have observations _at all_, it was purely a deterministic math sequence), because `Value` refers to reported values, which can have multiple measurements in-between. E.g. if we're reporting request count every second, but in the API we're calling `counter.Inc(1)` for every request, then I expect `Measurement_i == 1`, while `Value_j` are cumulative counts.

Reply:
> Does CUMULATIVE only applies to sums?

Good point. No, I don't think it does in theory. Likely in practice it does, but I'll change the sentence to be more general.

I don't want to let the discussion of `Sum`s distract from the underlying question you have about the sentence. To that end maybe we can use your example to make sure we share a common understanding of what a CUMULATIVE value is. To expand on it:

1. The system starts.
1. A request comes in, we call `counter.Inc(1)`.
1. Another request comes in, we call `counter.Inc(1)`.
1. Another request comes in, we call `counter.Inc(1)`.
1. The 1 second collection cycle ends. A metric is created for the export pipeline with a value of `3`.
1. A request comes in, we call `counter.Inc(1)`.
1. Another request comes in, we call `counter.Inc(1)`.
1. The 1 second collection cycle ends. A metric is created for the export pipeline with a value of `5` (`2` from this collection cycle, `3` from the previous totaling `5` (assuming a sum aggregation)).
1. A request comes in, we call `counter.Inc(1)`.
1. The 1 second collection cycle ends. A metric is created for the export pipeline with a value of `6` 

The value for the metric in this example is said to be CUMULATIVE. Does this sound correct to you?

If so is there a way you think we can rephrase the sentence to better describe this behavior?

If not, can you provide an revision of the above example that matches what you are thinking?

Reply:
If you want to generalize from sum, then it's an aggregation of all measurements from some fixed point in time up to the last reporting snapshot. And I think it's fine to provide a small example as well.

But I am still curious what aggregations other than a sum this could be used for.

Reply:
> If you want to generalize from sum, then it's an aggregation of all measurements from some fixed point in time up to the last reporting snapshot. And I think it's fine to provide a small example as well.

:+1: 

> But I am still curious what aggregations other than a sum this could be used for.

Currently there are not any alternate aggregations (nor do I anticipate there ever being). I have seen monitoring systems in the past measure cumulative averages and rates. Both of which are pretty useless IMO.

The thing I don't want to do is imply is that there is a predefined aggregation that should be assumed if the the `Temporality` is cumulative. That would incorrectly couple the concepts.

Reply:
Updated, PTAL.